.travis.yml

```diff
@@ -29,10 +29,12 @@ jobs:
       script:
         - make packages
         - make test-coverage
-
     - name: 'Integration Tests Linux'
       script:
         - make test-integration
+    - name: 'Regression Tests Linux'
```

Comment:
let's run in only on master in background. Or only by schedule if it's possible.

Reply:
As discussed on slack, I'm gonna remove it 👍.